### DISCUSSION TITLE
webchat support popup/popout

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [nioc/xmpp-web#129] [General]
I think this looks like a really nice client. However it is a bit to much (files) for what I need. I just want to add div to 1 existing website page. That looks like most chat support solutions[1]

Is there a possibility you can make a branch having this simple chat solution basic html/css/js?

[1]
![2](https://github.com/user-attachments/assets/ab434775-a7e3-498d-b2b6-a14bd2e1d665)

**Comment by nioc** [comment]
For adding a chat to an existing website, you should use something else like [ConverseJS ](https://conversejs.org/)

**Reply by f1-outsourcing** [reply]
No I already wrote with their developers, they said to create something.

**Reply by f1-outsourcing** [reply]
@nioc 
I would be willing to pay for some lightweight single function web support client. It can't be to much work to build something around these libraries[1] not? Or am I wrong with this?

https://github.com/nioc/xmpp-web/blob/master/src/services/

**Reply by f1-outsourcing** [reply]
What are the possibilities of converting your xmpp-web to a vanilla javascript webchat client. I only need this one window nothing else. I assume this should be not so complicated, or is it?
I am trying to find a developer for this, but they are all whining about doing this.
